vuepress/docs/docs/stitching.md

```diff
@@ -0,0 +1,200 @@
+# Stitching
+
+**Stitching** is a part of `caliban-tools` which can be used to use parts of another GraphQL API from an API you're bulding in situations where using Apollo federation isn't an option. You can also use it to fully subsume and proxy another GraphQL schema.
+
+In general, Federation should be your preferred choice.
+
+You should also be careful when using stitching since it's very easy to pull in large parts of an external schema's types and structure into your API. This can make it error prone since the likelihood that you'll get type clashes between your API and the API you're stitching in quickly increases.
+
+## Dependencies
+
+In order to use stitching, add `caliban-tools` to your dependencies:
+
+```scala
+libraryDependencies += "com.github.ghostdogpr" %% "caliban-tools" % "0.10.0"
```

Comment:
Can you make the version like here: https://github.com/ghostdogpr/caliban/commit/50ff7c52710e20b6e980ee7e0345356fc7b86036#diff-6ed0fca1c221d272e75a9f1bb6128476f384f16fe6a75231b0f6c86c873a46baR243

I will replace it after the next release, in the meantime people won't get confused about it 😄

Reply:
👍 0943a15.